src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -409,6 +410,13 @@ class ClusterResourceScheduler : public ClusterResourceSchedulerInterface {
   /// fields used.
   void FillResourceUsage(rpc::ResourcesData &resources_data) override;
 
+  /// Populate a UpdateResourcesRequest. This is inteneded to update the
+  /// resource totals on a node when a custom resource is created or deleted
+  /// (e.g. during the placement group lifecycle).
+  ///
+  /// \param Output parameter. Fills out all fields.
```

Comment:
Nit: Return docstring item instead of parameter.
```suggestion
  /// \return Filled out resource map.
```